src/main/java/org/fundacionjala/coding/richardseverich/Movies/Customer.java

```diff
@@ -0,0 +1,68 @@
+package org.fundacionjala.coding.richardseverich.Movies;
+
+import java.util.ArrayList;
+
+/**
+ * Created by RichardSeverich on 5/31/2017.
+ */
+public class Customer {
+    private String nameCustomer;
+    private ArrayList<Rental> rentalList;
+    private int totalAmount;
+    private int frequentRenterPoints;
+
+    /**
+     * @param name is name of costumer.
+     */
+    public Customer(String name) {
+        nameCustomer = name;
+        rentalList = new ArrayList<>();
+        totalAmount = 0;
+        frequentRenterPoints = 0;
+    }
+
+    /**
+     * @param arg is a rental.
+     */
+    public void addRental(Rental arg) {
+        rentalList.add(arg);
+    }
+
+    /**
+     * @return Returns the string to be displayed.
+     */
+    public String statement() {
```

Comment:
This method is doing 3 things, it is calculating the totalAmount, totalFrequentRenterPoints and it is printing.
Please split it.